src/dso_api/dynamic_api/routers.py

```diff
@@ -61,10 +62,40 @@ class DynamicAPIRootView(APIView):
 
     def get(self, request, *args, **kwargs):
         base = request.build_absolute_uri("/").rstrip("/")
+        datasets = list(Dataset.objects.filter(id__in=self.dataset_ids))
+
         return Response(
             {
                 "datasets": {
-                    id: base + reverse(f"dynamic_api:openapi-{id}") for id in self.dataset_ids
+                    ds.schema.id: {
+                        "id": ds.schema.id,
+                        "name": ds.name,
+                        "title": ds.schema.get("title", ""),
+                        "status": ds.schema.get("status", "Beschikbaar"),
+                        "description": ds.schema.get("description", ""),
+                        "api_type": "rest_json",
+                        "api_url": base + reverse(f"dynamic_api:openapi-{ds.schema.id}"),
+                        "documentation_url": base + f"/v1/docs/datasets/{ds.schema.id}.html",
```

Comment:
```suggestion
                        "documentation_url": f"{base}/v1/docs/datasets/{ds.schema.id}.html",
```

Als je toch f-strings gebruikt... ;-)